.github/workflows/s3.yml

```diff
@@ -0,0 +1,19 @@
+name: Upload Static Files
+
+on:
+  push:
```

Comment:
Can we change to trigger it when release? so we have time to check on dev environment